java/connectors/semantickernel-connectors-memory-postgresql/src/main/java/com/microsoft/semantickernel/connectors/memory/postgresql/PostgreSQLMemoryStore.java

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft. All rights reserved.
+package com.microsoft.semantickernel.connectors.memory.postgresql;
+
+import com.microsoft.semantickernel.connectors.memory.jdbc.JDBCMemoryStore;
+import com.microsoft.semantickernel.memory.MemoryStore;
+import reactor.core.publisher.Mono;
+
+import javax.annotation.Nonnull;
+import java.sql.Connection;
+import java.sql.SQLException;
+
+public class PostgreSQLMemoryStore extends JDBCMemoryStore {
+
+    public PostgreSQLMemoryStore() {
+        this.dbConnector = new PostgreSQLConnector();
+    }
+
+    public PostgreSQLMemoryStore(Connection connection) {
+        this();
+        this.dbConnection = connection;
+    }
+
+    public Mono<Void> connectAsync(@Nonnull String url) throws SQLException {
+        return super.connectAsync(url);
+    }
+
+    public static class Builder implements MemoryStore.Builder {
+        @Override
+        public MemoryStore build() {
+            return new PostgreSQLMemoryStore();
+        }
+        public MemoryStore build(Connection connection) {
+            return new PostgreSQLMemoryStore(connection);
+        }
```

Comment:
Is builder really necessary here? Is there something happening that cannot happen in a constructor?